training/distributed_training/pytorch/model_parallel_v2/shared-scripts/train_utils.py

```diff
@@ -124,9 +124,13 @@ def create_model(args, model_config, dtype, pretrained_model_weights=None):
     if pretrained_model_weights:
         _logger.info("Loading pretrained weights from %s.", pretrained_model_weights)
         if pversion.parse(transformers.__version__) < pversion.parse("4.37.1"):
-            model = AutoModelForCausalLM.from_pretrained(pretrained_model_weights)
+            model = AutoModelForCausalLM.from_pretrained(pretrained_model_weights, config=model)
```

Comment:
Should this be model_config instead of model?

Reply:
Thanks for the pointer. Updated